BoxContentSDK/BoxContentSDK/Clients/BOXContentClient.h

```diff
@@ -85,6 +85,12 @@ extern NSString *const BOXContentClientBackgroundTempFolder;
  */
 @property (nonatomic, nullable, readwrite, copy) NSDictionary *fetchTokenBlockInfo;
 
+/**
+ *  Used to set background requests' shouldPerformRequestImmediately provided those requests prepared
+ *  using BOXContentClient. E.g.: BOXFileRequest returned by fileInfoRequestWithID:associateID:
```

Comment:
This is a bit unclear... `...those request  prepared using`.  
Did you mean `... those request are built in BOXContentClient.`  ?  
I'm not getting the requirement from this comment.  
How can we enforce where the request comes from or how is it prepared?

Reply:
"Did you mean ... those request are built in BOXContentClient. ?" Yes

In normal cases, users use BOXContentClient categories e.g. BOXContentClient+File's - (BOXFileRequest *)fileInfoRequestWithID:(NSString *)fileID to create BOXFileRequest.

User can choose to instantiate BOXRequest directly e.g. [[BOXFileRequest alloc] initWithFileID:fileID] and configure other necessary properties for it to work similarly to how the above method works. This case should be rare. In this case, the request won't be affected by this property.